NHLBI_BioData_Catalyst/python/1_PICSURE_API_101.ipynb

```diff
@@ -2,844 +2,744 @@
+    "\n"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "4bf425c6",
    "metadata": {},
    "source": [
-    "A `dictionary` instance enables us to retrieve matching records by searching for a specific term, or to retrieve information about all the available variables, using the `find()` method. For instance, looking for variables containing the term `COPD` in their names is done this way: "
+    "We can also view additional information for individual variables using the `varInfo()` method. "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "id": "8db8ef93",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "dictionary = resource.dictionary()\n",
-    "dictionary_search = dictionary.find(\"COPD\")"
+    "first_var = my_variables.listPaths()[0]\n",
+    "my_variables.varInfo(first_var)"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "accf1dbf",
    "metadata": {},
    "source": [
-    "Subsequently, objects created by the `dictionary.find` method expose the search results via 4 different methods: `.count()`, `.keys()`, `.entries()`, and `.DataFrame()`. "
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "pprint({\"Count\": dictionary_search.count(), \n",
-    "        \"Keys\": dictionary_search.keys()[0:5],\n",
-    "        \"Entries\": dictionary_search.entries()[0:5]})"
+    "Now you can try to search for a term on your own. Below is sample code on how to search for the term `sex`. To practice searching the data dictionary, you can change \"sex\" to a term you are interested in. If there are results for your term, you will see them displayed in the convenient dataframe format using the `displayResults()` method."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "id": "73dd6b2b",
+   "metadata": {
+    "scrolled": true,
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "dictionary_search.DataFrame().head()"
+    "my_search = dictionary.find(\"sex\") # Change sex to be your term of interest\n",
+    "if len(my_search.listPaths()) > 0:\n",
+    "    my_search.displayResults()\n",
+    "else:\n",
+    "    print(\"Search term returned no results. Please try searching a different term.\")"
    ]
   },
   {
    "cell_type": "markdown",
+   "id": "a295cfe9",
    "metadata": {},
    "source": [
-    "**The `.DataFrame()` method enables us to get the result of the dictionary search in a pandas DataFrame format. This way, it allows us to:** \n",
-    "\n",
+    "## Using PIC-SURE to build a query and retrieve data\n",
+    "You can also use the PIC-SURE API to build a query and retrieve data. With this functionality, you can filter based on specific variables, add others, and export the data as a dataframe into this notebook. \n",
```

Comment:
small suggestion: Since there is no output while a query is working, maybe put a line here that says a query might take ~30 seconds-1 minute to complete? That way users know it is working and they should just wait